src/app.ts

```diff
@@ -93,7 +105,14 @@ export default async function newApp(): Promise<Express> {
   passport.use(bearerTokenStrategy());
   app.use(passport.initialize());
 
-  registerRoutes(app);
+  try {
+    await emailService.verifyTransport();
```

Comment:
is it possible to pass emailService as a parameter to newApp() ? this should simplify tests